t/01-test_needle.t

```diff
@@ -225,15 +225,44 @@ ok(!defined $res, "the contrast is just too different");
 
 $img1   = tinycv::read($data_dir . "xterm-started-20141204.test.png");
 $needle = needle->new("xterm-started-20141204.json");
-$res    = $img1->search($needle, 0, 0.7);
+($res, $cand) = $img1->search($needle, 0, 0.7);
 
-ok(defined $res, "xterm basically the same");
+ok(!defined $res, "xterm on GNOME is more blurry");
+is_deeply(
+    $cand->[0]->{area}->[1],
+    {
+        x          => 127,
+        w          => 39,
+        y          => 76,
+        h          => 18,
+        result     => 'fail',
+        similarity => '0.905881691408007'
+    },
+    'we find the xterm though'
+);
 
 $img1   = tinycv::read($data_dir . "pkcon-proceed-prompt-20141205.test.png");
 $needle = needle->new("pkcon-proceed-prompt-20141205.json");
-$res    = $img1->search($needle, 0, 0.7);
+($res, $cand) = $img1->search($needle, 0, 0.7);
+
+ok(!defined $res, "the prompt is the same to the human eye, but it differs in shades of gray");
+# 15.2 calculcates 92.9 and TW 92.8 - different opencv versions, different compilers...
```

Comment:
The 15.2 is a bit ambiguous. Could you please rephrase? Either "openSUSE Leap 15.2" or just different OS versions.